### ISSUE TITLE
Style/FetchEnvVar should not warn on method call with safe navigation

### ISSUE THREAD

**Issue by jcoyne** [rubocop/rubocop#10579]
Currently `ENV["VAR"].some_method` is permitted.  However the cop triggers if you use the safe navigation operator to call that method. I don't think it should.

So that would allow something like this:

```ruby
ENV['ROLES']&.split(';')
```
